test/core/parser/grammar/grammar_sequence_test.py

```diff
@@ -492,17 +474,25 @@ def test__parser__grammar_bracketed_modes(
 
 
 @pytest.mark.parametrize(
-    "input_seed,mode,sequence",
+    "input_seed,mode,sequence,kwargs",
     [
-        # Unclosed brackets always raise errors.
-        (["(", "a"], ParseMode.STRICT, ["a"]),
-        (["(", "a"], ParseMode.GREEDY, ["a"]),
+        # Unclosed greedy brackets always raise errors.
+        (["(", "a"], ParseMode.GREEDY, ["a"], {}),
+        (
+            ["(", " ", ")"],
+            ParseMode.GREEDY,
+            [],
+            # Greedy matching, without allowing gaps, should raise a parsing error.
+            # NOTE: This functionality doesn't get used much.
+            {"allow_gaps": False},
+        ),
```

Comment:
👍 I think I'm good with this behaviour change. It's not used, and as long as the other unparsable cases work, then I think we're good.